### ISSUE TITLE
[Bug]: ImportError: DLL load failed while importing _path:

### ISSUE THREAD

**Issue by a-sajjad72** [matplotlib/matplotlib#25002]
### Bug summary

```
from matplotlib._path import (
ImportError: DLL load failed while importing _path: The specified module could not be found.```

got error on freshly installed windows 10 22h2.

### Code for reproduction

```python
just import the module
`import matplotlib`
```

### Actual outcome

>>> import matplotlib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\__init__.py", line 113, in <module>
    from . import _api, _version, cbook, _docstring, rcsetup
  File "C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\rcsetup.py", line 27, in <module>
    from matplotlib.colors import Colormap, is_color_like
  File "C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\colors.py", line 56, in <module>
    from matplotlib import _api, _cm, cbook, scale
  File "C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\scale.py", line 22, in <module>
    from matplotlib.ticker import (
  File "C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\ticker.py", line 138, in <module>
    from matplotlib import transforms as mtransforms
  File "C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\transforms.py", line 49, in <module>
    from matplotlib._path import (
ImportError: DLL load failed while importing _path: The specified module could not be found.

### Expected outcome

the module should be successfully imported.

### Operating system

Windows 10 Pro Version	22H2

### Matplotlib Version

matplotlib-3.6.3

### Python version

Python 3.10.8

### Installation

pip

**Comment by oscargus** [comment]
Do you have a file called something like `_path.cp310-win32.pyd` in `C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib`?

**Comment by a-sajjad72** [comment]
yes, there is a file with the name `_path.cp310-win32.pyd` in `C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\Lib\site-packages\matplotlib`

**Comment by a-sajjad72** [comment]
> Do you have a file called something like `_path.cp310-win32.pyd` in `C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib`?

yes, there is a file with the name `_path.cp310-win32.pyd` in `C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\Lib\site-packages\matplotlib`

**Comment by QuLogic** [comment]
Can you run Dependency Walker on the DLL to find out which runtime DLL is missing?

**Comment by a-sajjad72** [comment]
> Do you have a file called something like `_path.cp310-win32.pyd` in `C:\Users\HaLF-MenTaL\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib`?

i again checked the file you mentioned `_path.cp310-win32.pyd`
sorry, i was mistaken the file which I have is `_path.cp310-win_amd64.pyd` and it's `win_amd64` not `win32`. let me confirm is it the same file you want or another one? in addition mine os is x64. the filename vary with architecture and i think it's the same file which you asked. is it???

**Comment by oscargus** [comment]
That is the one, yes. I just guessed which architecture you used...

**Comment by a-sajjad72** [comment]
> Can you run Dependency Walker on the DLL to find out which runtime DLL is missing?

[here](https://gist.github.com/a-sajjad72/abb0ca9d5c99c8412956f7e6c3857814) are the logs generated by dependency walker

**Comment by QuLogic** [comment]
This is a duplicate of #24704; I thought we had backported the fix to 3.6.3, but we didn't. It should be fixed in 3.7.0. You should be able to get it working by installing `pip install msvc-runtime`.
